app/controllers/appointments_controller.rb

```diff
@@ -59,7 +59,7 @@ def search_params # rubocop:disable Metrics/MethodLength
   def notify_customer(appointment)
     return unless appointment.notify?
 
-    Appointments.customer(appointment, booking_location.object).deliver_later
+    AppointmentChangeNotificationJob.perform_later(appointment)
```

Comment:
A lot of things happen here. Can we test higher up the overall effects of this call? Take a look at the feature specs and do a quick assertion there that this job is enqueued or preferably that a mail is sent and the associated activity is created.